src/timer.c

```diff
@@ -30,17 +30,17 @@
 #include "notes-area.h"
 #include "prefs.h"
 #include "proj.h"
+#include "projects-tree.h"
 #include "props-task.h"
 #include "timer.h"
-#include "projects-tree.h"
 
 int config_autosave_period = 60;
-int config_autosave_props_period = (4*3600);
+int config_autosave_props_period = (4 * 3600);
 
-static gint main_timer = 0;
-static gint file_save_timer = 0;
-static gint config_save_timer = 0;
-static GttIdleDialog *idle_dialog = NULL;
+static gint             main_timer = 0;
+static gint             file_save_timer = 0;
+static gint             config_save_timer = 0;
+static GttIdleDialog   *idle_dialog = NULL;
```

Comment:
Is the idea here to realign inside each line if one adds a new variable with a longer type name that doesn't fit?

I suppose it could lead to increasing size of diffs by adjusting lines that didn't otherwise need semantic changes. Though perhaps it's manageable. I haven't used this style in any project I've worked extensively on so I can't tell if that tend to happen a lot or not.

I do see some value in increased readability so I'm not against it.

Reply:
On second thought I'm really not happy with this. Some of the exiting code did this and I tried to minimize the initial diff. On one hand it _can_ improve readability, but it looks odd if the types are very different in length. Your point about needlessly increasing diffs is a major one too. I disabled the alignment of declarations now.